.gitignore

```diff
@@ -4,3 +4,4 @@ _build
 _opam
 .envrc
 shell.nix
+*~
```

Comment:
What are these, editor temp files?

Reply:
Suffix for backup files for almost old traditional UNIX text editors, including Emacs, Vim and nano, AFAIK.